src/System.Private.CoreLib/src/System/Reflection/Emit/DynamicILInfo.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed to the.NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Reflection.Emit
+{
+    public class DynamicILInfo
+    {
+        public DynamicMethod DynamicMethod { get { return default; } }
+
+        public void SetCode(byte[] code, int maxStackSize)
+        {
+            ReflectionEmitThrower.ThrowPlatformNotSupportedException();
```

Comment:
You do not really need the instance methods to throw. They can be no-ops (it is what they are in other similar files) since we do not allow the instance to be constructed in the first place.